scripts/run_nodes.sh

```diff
@@ -65,7 +65,9 @@ echo "Bootstrapping chain for nodes 0..$((N_VALIDATORS - 1))"
 for i in $(seq "$N_VALIDATORS" "$(( N_VALIDATORS + N_NON_VALIDATORS - 1 ))"); do
   echo "Bootstrapping node $i"
   account_id=${account_ids[$i]}
-  ./target/release/aleph-node bootstrap-node --base-path "$BASE_PATH" --account-id "$account_id" --chain-type local
+  unit_saving_path="$BASE_PATH/$account_id/unit-stash"
+  [ -e "$unit_saving_path" ] || mkdir "$unit_saving_path"
+  ./target/release/aleph-node bootstrap-node --base-path "$BASE_PATH" --unit-saving-path "$unit_saving_path" --account-id "$account_id" --chain-type local
```

Comment:
This needs changes to bootstrap-node command in `bin/node/src/commands.rs`. Also `bootstrap-chain` needs to be alterated a little. Stash should be removed on each run. `docker-runtime-hook/entrypoint.sh` also needs some alteration, to add backup path.